### ISSUE TITLE
Why use HAVE_VAAPI_DRM in the libyami.cpp 

### ISSUE THREAD

**Issue by zackxue** [01org/ffmpeg_libyami#28]
The HAVE_VAAPI_DRM enable use the DRM instead of X11 ? can i change it to X11?

**Comment by mypopydev** [comment]
@zackxue Yes, you can use X11, but I think you need to change the code and re-build the ffmpeg_libyami source code, another question, why you prefer X11 than DRM?

**Comment by zackxue** [comment]
Refer  https://github.com/01org/ffmpeg_libyami/issues/23, the AV_PIX_FMT_YAMI  default is DRM, can not putsurface.
